documentation/src/docs/asciidoc/user-guide/extensions.adoc

```diff
@@ -715,6 +715,21 @@ test methods.
 include::{testDir}/example/exception/MultipleHandlersTestCase.java[tags=user_guide]
 ----
 
+[[extensions-preinterrupt-callback]]
+=== PreInterrupt Callback
+
+`{PreInterruptCallback}` defines the API for `Extensions` that wish to react on
```

Comment:
For this to work, please add an entry to `documentation/src/docs/asciidoc/link-attributes.adoc`:
```asciidoc
:PreInterruptCallback:                       {javadoc-root}/org.junit.jupiter.api/org/junit/jupiter/api/extension/PreInterruptCallback.html[PreInterruptCallback]
```